src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CreateProbationBookingService.kt

```diff
@@ -51,8 +51,7 @@ class CreateProbationBookingService(
       probationTeam = probationTeam,
       probationMeetingType = request.probationMeetingType!!.name,
       comments = request.comments,
-      createdBy = createdBy.username,
-      createdByPrison = false,
```

Comment:
Noticed this bug during testing.  In the new world prison users can create probation bookings.